frontend/src/features/entity_selector/EntityTabsBox.tsx

```diff
@@ -43,11 +108,21 @@ const EntityTabsBox = ({
   maxHeight = '40vh',
   withLink = false,
   entityTextInput,
+  displayDescription = false,
 }: Props) => {
   const { t } = useTranslation();
+
   const [tabValue, setTabValue] = useState(tabs[0]?.name);
   const [status, setStatus] = useState<TabStatus>(TabStatus.Ok);
   const [options, setOptions] = useState<RelatedEntityObject[]>([]);
+  const [toggleDescription, setToggleDescription] =
+    useState(displayDescription);
```

Comment:
(minor)
Using "toggle" to name the state sounds confusing to me. What about something like `isDescriptionVisible` ?

```suggestion
  const [isDescriptionVisible, setIsDescriptionVisible] =
    useState(displayDescription);
```